Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverSurfaceMagnetism.cs

```diff
@@ -90,10 +90,8 @@ public enum OrientModeEnum
         private const float maxDot = 0.97f;
         #endregion
 
-        protected override void Start()
+        protected void Start()
```

Comment:
No changes to Solver.cs did we remove the base start method from it?

Reply:
`Start` was removed from ControllerFinder